gradle/libs.versions.toml

```diff
@@ -80,8 +80,8 @@ decompose = "3.2.2"
 diffplug-spotless = "6.5.2"
 dropwizard-metrics = "4.2.26"
 eclipse-ecj = "3.39.0"
-eclipse-jetty = "10.0.22"
-eclipse-jettytoolchain = "4.0.6"
+eclipse-jetty = "12.0.10"
```

Comment:
Should probably go to 12.0.19 - https://github.com/jetty/jetty.project/releases/tag/jetty-12.0.19